cleanlab/datalab/issue_finder.py

```diff
@@ -69,8 +70,9 @@ class IssueFinder:
     `Datalab.find_issues` method which internally utilizes an IssueFinder instance.
     """
 
-    def __init__(self, datalab: "Datalab", verbosity=1):
+    def __init__(self, datalab: "Datalab", imagelab: "Imagelab", verbosity=1):
         self.datalab = datalab
+        self.imagelab = imagelab
```

Comment:
No need to pass `imagelab` as a parameter to the constructor. It's already a field of `datalab`.